content/en/blog/2024/spring-starter-stable/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: The OpenTelemetry Spring Boot starter is now stable
+linkTitle: Spring Starter GA
+date: 2024-
```

Comment:
```suggestion
date: 2024-
```

@zeitlinger any timelines for publishing this?

Reply:
in about a month - if possible I'd like to have the micrometer explanation be published before 

@trask fyi